alvin/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wordle</title>
+
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+    <h1>poor man wordle unlimited </h1>
+    <hr>
+    <div id="rules">
+        <h2>Rules</h2>
+        <h3>If you haven't played wordle.. well you just picked the poor mans version but at least we have a new game button :D</h3>
+        <ul>
+            <li>If the tile is green, the letter is correct and in the right place.</li>
+            <li>If the tile is yellow, the letter is correct but in the wrong place.</li>
+            <li>If the tile is grey, man you gotta guess another word.</li>
+
```

Comment:
```suggestion
```

Though browser defaults to collapsing whitespaces, it's a good idea to use indentations and newlines to group children elements.